libs/sdk-ui-kit/src/SettingItem/tests/SettingItem.test.tsx

```diff
@@ -0,0 +1,59 @@
+// (C) 2022 GoodData Corporation
+import React from "react";
+import { render, screen, waitFor } from "@testing-library/react";
+// import userEvent from "@testing-library/user-event";
```

Comment:
Please delete this line rather than commenting it out.

Reply:
fixed